packages/codium.rb

```diff
@@ -3,17 +3,17 @@
 class Codium < Package
   description 'VSCodium is Open Source Software Binaries of VSCode with a community-driven default configuration.'
   homepage 'https://vscodium.com/'
-  version '1.64.1'
+  version '1.64.2'
   license 'MIT'
   compatibility 'aarch64,armv7l,x86_64'
   case ARCH
   when 'aarch64', 'armv7l'
     source_url "https://github.com/VSCodium/vscodium/releases/download/#{version}/VSCodium-linux-armhf-#{version}.tar.gz"
```

Comment:
```ruby
source_url ({
  aarch64: "https://github.com/VSCodium/vscodium/releases/download/#{version}/VSCodium-linux-armhf-#{version}.tar.gz",
  armv7l: "https://github.com/VSCodium/vscodium/releases/download/#{version}/VSCodium-linux-armhf-#{version}.tar.gz",
  x86_64: "https://github.com/VSCodium/vscodium/releases/download/#{version}/VSCodium-linux-x64-#{version}.tar.gz"
})

source_sha256 ({
  aarch64: '5594b22a6f624d5b99f710d4b05f135e0b54a7eb13179f3fe42614b514887e15',
  armv7l: '5594b22a6f624d5b99f710d4b05f135e0b54a7eb13179f3fe42614b514887e15',
  x86_64: '682ea0170d5e533dd2bd93f8e2deaad826339a1e46bea6e4be362d7b32b9ae51'
})
```

Reply:
In most other packages that would work but in this case we have the `@arch` variable assignment.